### PR TITLE
Delivery verification article creation

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-a8d558cd36/how-to/)

**Comment by stephamon** [comment]
@kmsholcomb I've got one more! Don't worry about the spam filtering one. That's in another PR that I tagged you in already. It got included in this one by mistake :)

**Comment by kmsholcomb** [comment]
Hi @stephamon. My suggested edits for the email delivery article are attached. Please let me know if you have any questions. Thanks!

[Verify email delivery_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1052910/Verify.email.delivery_editsKH.pdf)
[Verify email delivery_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1052911/Verify.email.delivery_editsKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-14dbc3b41e/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-cc67297ee5/how-to/).

File: content/rackspace-email/spam-filtering.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
+
```
Comment (DonNateR) [inline content/rackspace-email/spam-filtering.md]: We should delete the extra line here.

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-a75a370607/how-to/)

**Pull request merged**